modules/s1/main.tf

```diff
@@ -0,0 +1,570 @@
+provider "aws" {
+  region = var.region
+}
+
+locals {
+  api_domain_name = join(".", [var.api_subdomain, var.domain_name])
+}
+
+### VPC
+module "vpc" {
+  source = "terraform-aws-modules/vpc/aws"
+
+  enable_ipv6 = true
+
+  name = join("-", [var.env_name, var.region, "vpc"])
+  cidr = "10.0.0.0/16"
+
+  azs             = ["eu-central-1a", "eu-central-1b", "eu-central-1c"]
+  private_subnets = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
+  public_subnets  = ["10.0.101.0/24", "10.0.102.0/24", "10.0.103.0/24"]
+
+  enable_nat_gateway     = true
+  single_nat_gateway     = true
```

Comment:
Can we clarify if we actually need NAT gateways? We have them in S3 but they are our biggest expense and I'm not sure we actually need them...

Reply:
I don't think so either with the API gateway and lambda infrastrcture. I will remove them